src/main/java/org/opentripplanner/routing/core/intersection_model/SimpleIntersectionTraversalCostModel.java

```diff
@@ -37,11 +37,7 @@ public double computeTraversalCost(
             return computeDrivingTraversalCost(v, from, to, request);
         }
         else if (mode.isCycling()) {
-            var c = computeCyclingTraversalCost(v, from, to, fromSpeed, toSpeed, request);
-            if (LOG.isTraceEnabled()) {
-                LOG.trace("Turning from {} to {} has a cost of {}", from, to, c);
-            }
-            return c;
+            return computeCyclingTraversalCost(v, from, to, fromSpeed, toSpeed, request);
```

Comment:
Unrelated?

Reply:
It actually showed up in the profiler as costing some performance, so I deleted it. It effect is probably small but every millisecond counts, doesn't it?